backends/ubpf/midend.cpp

```diff
@@ -79,27 +79,26 @@ const IR::ToplevelBlock *MidEnd::run(EbpfOptions &options, const IR::P4Program *
              new P4::RemoveMiss(&refMap, &typeMap), new P4::ClearTypeMap(&typeMap),
              new P4::EliminateNewtype(&refMap, &typeMap),
              new P4::EliminateInvalidHeaders(&refMap, &typeMap),
-             new P4::SimplifyControlFlow(&refMap, &typeMap),
+             new P4::SimplifyControlFlow(&typeMap),
              new P4::SimplifyKey(
                  &refMap, &typeMap,
                  new P4::OrPolicy(new P4::IsValid(&refMap, &typeMap), new P4::IsLikeLeftValue())),
              new P4::ConstantFolding(&refMap, &typeMap),
              // accept non-constant keysets
              new P4::SimplifySelectCases(&refMap, &typeMap, false), new P4::HandleNoMatch(&refMap),
              new P4::SimplifyParsers(&refMap),
-             new PassRepeated({new P4::ConstantFolding(&refMap, &typeMap),
-                               new P4::StrengthReduction(&refMap, &typeMap)}),
+             new PassRepeated(
+                 {new P4::ConstantFolding(&refMap, &typeMap), new P4::StrengthReduction(&typeMap)}),
              new P4::SimplifyComparisons(&refMap, &typeMap),
              new P4::CopyStructures(&refMap, &typeMap),
              new P4::LocalCopyPropagation(&refMap, &typeMap),
              new P4::SimplifySelectList(&refMap, &typeMap),
              new P4::MoveDeclarations(),  // more may have been introduced
              new P4::RemoveSelectBooleans(&refMap, &typeMap),
              new P4::SingleArgumentSelect(&refMap, &typeMap),
-             new P4::ConstantFolding(&refMap, &typeMap),
-             new P4::SimplifyControlFlow(&refMap, &typeMap), new P4::TableHit(&refMap, &typeMap),
-             new P4::RemoveLeftSlices(&refMap, &typeMap), new EBPF::Lower(&refMap, &typeMap),
-             evaluator, new P4::MidEndLast()});
+             new P4::ConstantFolding(&refMap, &typeMap), new P4::SimplifyControlFlow(&typeMap),
+             new P4::TableHit(&refMap, &typeMap), new P4::RemoveLeftSlices(&refMap, &typeMap),
+             new EBPF::Lower(&refMap, &typeMap), evaluator, new P4::MidEndLast()});
```

Comment:
Can you break these passes out into separate lines for readability please? Unless clang-format is always pulling them back into the structure you have.

Reply:
> Another breaking change 😱 😉

Sorry :) Yes, I feel your pain :) I already ported some decent amount of downstream code for this as well...

Let me see if clang-format could be justified to put these on separate lines as this formatting is all from it.

Reply:
Usually there is a comma missing at the end of a list. This is an undocumented clang-format feature.

Reply:
@fruffy Yes, this is exactly this

Reply:
This is done